README.md

```diff
@@ -55,7 +55,7 @@ and import it into File→Settings→Editor→Code Style.
 
 ### Eclipse
 
-A
+A (old) 
```

Comment:
Could you instead say something like, "Version 1.6 of the [google-java-format Eclipse Plugin]..."?

Reply:
Fix in 351b737be5580c2f3320605668c218882624a3fc